bobtemplates/plone/viewlet/viewlets/+viewlet_name_normalized+.py.bob

```diff
@@ -0,0 +1,14 @@
+from plone.app.layout.viewlets import ViewletBase
+from datetime import datetime
+
+
+CONFERENCE_START_DATE = datetime(2018, 11, 5)
```

Comment:
you may use `datetime.now()` or `datetime.now().date()` instead of hardcoding  a starting date

Reply:
I removed it in the latest commit check [this](https://github.com/plone/bobtemplates.plone/blob/aa6c1a7bbe03560a355888f714d42e0774e9f885/bobtemplates/plone/viewlet/viewlets/%2Bviewlet_name_normalized%2B.py.bob)